la.ogri.strongbox.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>la.ogri.strongbox</id>
+    <name>strongbox</name>
+
+    <keywords>
+        <keyword>warcraft</keyword>
+        <keyword>world of warcraft</keyword>
+        <keyword>addon manager</keyword>
+    </keywords>
+
+    <project_license>AGPL-3.0-only</project_license>
+    <metadata_license>CC0-1.0</metadata_license>
+
+    <summary>strongbox is an addon manager for World of Warcraft.</summary>
+    <description>
+        <p>
+            strongbox is an open source, advertisement free and privacy respecting addon manager for World of Warcraft.
+        </p>
+    </description>
+
+    <launchable type="desktop-id">strongbox.desktop</launchable>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/ogri-la/strongbox/develop/screenshots/screenshot-7.0.0-installed.png</image>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/ogri-la/strongbox/develop/screenshots/screenshot-7.0.0-dark-installed.png</image>
```

Comment:
Pull screenshots from a tag/commit instead of a branch

Reply:
fixed